packages/okta-auth-js/webpack.dev.config.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
I'm not sure what this file is doing. 

The only real change I see is the `watch` flag, which can be done with `yarn build --watch`

Is it trying to avoid some setting in `webpack.config.js` ?

Reply:
It also removed uglify plugin, make it easier for local debugging.